source/sass/formassembly-override.scss

```diff
@@ -0,0 +1,147 @@
+@import "mofo-bootstrap/colors";
```

Comment:
Using SCSS for overrides instead of applying Tailwind classes directly to DOM elements. This way we won't have to re-apply classes every time we get a new version of code snippets from FA.